mungegithub/mungegithub.go

```diff
@@ -25,12 +25,13 @@ import (
 	"k8s.io/contrib/mungegithub/features"
 	github_util "k8s.io/contrib/mungegithub/github"
 	"k8s.io/contrib/mungegithub/mungers"
+	"k8s.io/contrib/mungegithub/mungers/mungerutil"
 	"k8s.io/contrib/mungegithub/reports"
 	utilflag "k8s.io/kubernetes/pkg/util/flag"
 
+	"github.com/foxish/viper"
```

Comment:
@eparis Sure, I can do that. However, @lavalamp made a good point that the huge number of dependencies we're adding for relatively simple reading of a config YAML might not be worth it. So, I think I might move away to reading the yaml ourselves in this case, unless there is a good reason to stick with viper.